src/schemas/documentSchema.ts

```diff
@@ -8,6 +8,11 @@ export const documentSchema = yup
     type: yup.string().required(),
     title: yup.string().required(),
     source_url: yup.string().url().required(),
-    user_language_name: yup.string().optional(),
+    user_language_name: yup
+      .object({
+        label: yup.string().required(),
```

Comment:
This doesn't match the backend DTOs, which say that user_language_name is optional. Can someone advise? @diversemix @alaniswright